app/views/coordinate.scala

```diff
@@ -69,8 +69,9 @@ object coordinate {
           )
         ),
         div(cls := "coord-trainer__board main-board")(
-          div(cls := "next_coord", id := "next_coord0"),
-          div(cls := "next_coord", id := "next_coord1"),
+          raw(
+            """<svg class="coords-svg" viewBox="0 0 100 100"><text class="coord current-coord"/><text class="coord next-coord"/></svg>"""
+          ),
```

Comment:
Any reason not to use scalatags here? If there aren't any predefined tags for `svg`, `text`, or `viewBox` you can just use `tag("svg")` or `attr("viewBox")` to add them here: https://github.com/ornicar/lila/blob/master/app/ui/scalatags.scala

Reply:
No special reason for that, just took the only svg usage in scala templates I've found in the project (`spinner` in [Environment.scala](https://github.com/ornicar/lila/blob/master/app/templating/Environment.scala)) as an example.
Thanks for the tip - will rework with `tag` and `attr`.

Reply:
Done: d130cfeaf9692505bbb90245aa3fcab31b603834

Reply:
Done